frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/install.tsx

```diff
@@ -59,6 +68,17 @@ export const CreateAttachedDevicesCluster: React.FC<CreateAttachedDevicesCluster
     setLsoSubscription(subscriptions.find((item) => item?.spec?.name === LSO_OPERATOR));
   }, [subscriptions]);
 
+  React.useEffect(() => {
+    k8sList(StorageClassModel)
+      .then((storageClasses: StorageClassResourceKind[]) => {
+        const filteredSCData = storageClasses.filter(filterSCWithNoProv);
+        if (filteredSCData.length) {
+          setHasNoProvSC(true);
+        }
+      })
+      .catch(() => setHasNoProvSC(false));
+  }, [appName, ns]);
+
```

Comment:
Use `useK8sGet` and move this to the wizard component. We can directly use this in child component.